Zend/zend_types.h

```diff
@@ -236,17 +247,7 @@ typedef struct _Bucket {
 typedef struct _zend_array HashTable;
 
 struct _zend_array {
-	zend_refcounted_h gc;
-	union {
-		struct {
-			ZEND_ENDIAN_LOHI_4(
-				zend_uchar    flags,
-				zend_uchar    _unused,
-				zend_uchar    nIteratorsCount,
-				zend_uchar    consistency)
```

Comment:
We probably may use two low bits of extra_flags to implement consistency checks (now you commented them)